{
  "results": {
    "dockerfile": {
      "exist": false,
      "count": 0,
      "rootCount": 0,
      "subFolderCount": 0,
      "filepaths": []
    },
    "composefile": {
      "exist": false,
      "count": 0,
      "rootCount": 0,
      "subFolderCount": 0,
      "filepaths": []
    },
    "kustomize": {
      "exist": false,
      "count": 0,
      "rootCount": 0,
      "subFolderCount": 0,
      "filepaths": []
    },
    "helm": {
      "exist": false,
      "count": 0,
      "rootCount": 0,
      "subFolderCount": 0,
      "filepaths": []
    },
    "k8s": {
      "exist": false,
      "count": 0,
      "rootCount": 0,
      "subFolderCount": 0,
      "filepaths": []
    },
    "backstage": {
      "exist": false,
      "count": 0,
      "rootCount": 0,
      "subFolderCount": 0,
      "filepaths": []
    },
    "terraform": {
      "exist": false,
      "count": 0,
      "rootCount": 0,
      "subFolderCount": 0,
      "filepaths": []
    }
  },
  "info": {
    "id": 137429940,
    "name": "pytorch-deeplab-xception",
    "full_name": "jfzhang95/pytorch-deeplab-xception",
    "owner": {
      "login": "jfzhang95",
      "html_url": "https://github.com/jfzhang95",
      "type": "User"
    },
    "html_url": "https://github.com/jfzhang95/pytorch-deeplab-xception",
    "description": "DeepLab v3+ model in PyTorch. Support different backbones.",
    "url": "https://api.github.com/repos/jfzhang95/pytorch-deeplab-xception",
    "clone_url": "https://github.com/jfzhang95/pytorch-deeplab-xception.git",
    "homepage": "",
    "size": 939,
    "stargazers_count": 2686,
    "watchers_count": 2686,
    "language": "Python"
  }
}